imports/ui/components/Card.jsx

```diff
@@ -108,13 +112,18 @@ class CardComponent extends Component {
                   <Typography variant="subtitle1" color="textSecondary">
                     ${this.state.data.price} {this.state.data.unit}
                   </Typography>
-                  <Typography
-                    variant="subtitle2"
-                    className={classes.longAddress}
-                    color="textSecondary"
+                  <Tooltip
```

Comment:
I think this tooltip can wrap the whole details section (Item name, price, address) instead, because that whole part can be clicked tooltip will useful in that area and not just when hovered on address.

Reply:
Here's what it looks like when wrapping the whole Card Content (with placement on bottom)
![Screen Shot 2019-07-29 at 3 27 29 PM](https://user-images.githubusercontent.com/25490855/62087364-5238f580-b216-11e9-9c03-ad548728ea5c.png)

Here's what placement on top looks like
![Screen Shot 2019-07-29 at 3 33 21 PM](https://user-images.githubusercontent.com/25490855/62087384-641a9880-b216-11e9-8b45-e2527b825bdf.png)

This is due to our component taking up the whole card space, and the set distance between tooltip and the component. 
![Screen Shot 2019-07-29 at 3 27 17 PM](https://user-images.githubusercontent.com/25490855/62087412-78f72c00-b216-11e9-89ef-572219abc787.png)

Here's what it looks like if I set the tooltip to appear only on the "blue section"
![Screen Shot 2019-07-29 at 3 41 53 PM](https://user-images.githubusercontent.com/25490855/62087698-852fb900-b217-11e9-8ee7-1181a0a7a695.png)
![Screen Shot 2019-07-29 at 3 42 14 PM](https://user-images.githubusercontent.com/25490855/62087699-85c84f80-b217-11e9-8541-8f5ad3d605a6.png)

Thoughts?